src/scripts/browserDeviceProfile.js

```diff
@@ -1156,14 +1156,32 @@ export default function (options) {
             })
         };
 
-        codecProfile.Conditions.push({
+        codecProfileTS.Conditions.push({
             Condition: 'LessThanEqual',
             Property: 'VideoLevel',
             Value: '42',
             IsRequired: false
         });
 
-        profile.CodecProfiles.push(codecProfile);
+        profile.CodecProfiles.push(codecProfileTS);
+
+        const codecProfileMp4 = {
+            Type: 'Video',
+            Codec: 'h264',
+            Container: 'mp4',
```

Comment:
You can omit `Container` if you need to apply the condition to H264 in any container.

Reply:
Indeed. If streaming through HLS fails, directplay may fail too. Directplay is usually more strict on Safari.

Reply:
Same video direct plays in an mov container (without this patch)